src/server/core/config.ts

```diff
@@ -56,7 +57,8 @@ export function readConfiguration(dir: string, file: string): ConfigurationFile
           return config;
         }
       } catch (e) {
-        console.error(`Error reading configuration from ${file} in ${dir}: ${e}`);
+        const msg = `Error reading configuration from ${file} in ${dir}: ${e}`;
+        throw new Error(`${chalk.red('ERR')} ${chalk.white(msg)}`);
```

Comment:
This is a breaking change in behavior.

It's fine in general, but the version should be adopted for it.

Reply:
In order to make it easier for users to see errors, when kras fails to start properly due to incorrect configuration files.

Reply:
I know. Still its a breaking change.

As written you need to change the version (and add it to the changelog) in your PR. There is no way around it.

Reply:
OK